docs/src/main/sphinx/release/release-469.md

```diff
@@ -1,89 +1,175 @@
+* Improve stability of the cluster under load. ({issue}`24572`) 
+* Refrain from join pushdown for modified tables. ({issue}`24447`)
+* Fix parsing of negative `0x`, `0b`, `0o` long literals. ({issue}`24601`)
 
 ## Web UI
 
-## JDBC driver
+* Add support for filtering by query client tags. ({issue}`24494`)
+* Add authentication support for the Preview Web UI with the authentication
+  types `FORM`, `FIXED`, `OAUTH2`, `JWT`, `KERBEROS`, `CERTIFICATE`, and
+  `INSECURE`. ({issue}`23230`)
+* Sort queries on the current worker status section in the Preview Web UI by
+  descending reserved memory. ({issue}`24475`)
 
-## Server RPM
+## JDBC driver
 
-## Docker image
+* Add `planningTimeMillis`, `analysisTimeMillis`, `finishingTimeMillis`,
+  `physicalInputBytes`, `physicalWrittenBytes`, `internalNetworkInputBytes` and
+  `physicalInputTimeMillis` to `io.trino.jdbc.QueryStats`. ({issue}`24571`,
+  {issue}`24604`)
+* Add the `validateConnection` connection property to allows validating
+  connection, and improve `Connection.isValid(int)` so it validates connection
+  and credentials. ({issue}`24127`, {issue}`22684`)
+* Fix loading spooled segment data when cluster is secured. ({issue}`24595`)
+* Fix deserialization failures with `SetDigest`, `BingTile`, and `Color` types. ({issue}`24612`)
 
 ## CLI
 
-## BigQuery connector
+* Fix loading spooled segment data when cluster is secured. ({issue}`24595`)
+* Fix deserialization of `SetDigest`, `BingTile`, and `Color` types. ({issue}`24612`)
 
-## Blackhole connector
+## BigQuery connector
 
-## Cassandra connector
+* Allow configuration of the channel pool for gRPC communication with BigQuery. ({issue}`24638`)
 
 ## ClickHouse connector
 
-## Delta Lake connector
-
-## Druid connector
+* {{breaking}} Raise minimum required versions to ClickHouse 24.3 and Altinity
+  22.3. ({issue}`24515`)
+* Fix failure when updating values to `NULL`. ({issue}`24204`)
 
-## Elasticsearch connector
+## Delta Lake connector
 
-## Exasol connector
+* Add support for SSE-C in S3 security mapping. ({issue}`24566`)
+* Allow configuring the endpoint for the Google Storage file system with the
+  `gcs.endpoint` property. ({issue}`24626`)
+* Improve performance of reading from new Delta Lake table data by compressing
+  files with `ZSTD` by default, instead of the previous `SNAPPY`.
+  ({issue}`17426`)
+* Improve performance of queries on tables with large transaction log JSON
+  files. ({issue}`24491`)
+* Improve performance of reading from Parquet files with a large number of row
+  groups. ({issue}`24618`)
+* Improve performance for the `OPTIMIZE` statement by enabling concurrent
+  execution. ({issue}`16985`)
+* Improve performance of reading from large files on S3. ({issue}`24521`)
+* Fix JMX metrics for file system caching with multiple catalogs. ({issue}`24510`)
+* Fix reading from tables when using the Alluxio file system. ({issue}`23815`)
+* Fix incorrect results when writing with deletion vectors. ({issue}`24648`)
```

Comment:
What does “writing with deletion vectors” mean?

Reply:
I reworded both entries around deletion vectors .. maybe @ebyhr and @findinpath can confirm .. essentially deletion vectors is a feature that you can enable on Delta Lake tables.. and when you do that .. there are scenario where writing as well as reading data from the table is wrong like wrong row counts and such